crypto-jvm/src/main/java/care/data4life/crypto/GCSymmetricKey.kt

```diff
@@ -16,6 +16,8 @@
 
 package care.data4life.crypto
 
-import com.squareup.moshi.Json
+import care.data4life.crypto.security.SecretKey
+import com.squareup.moshi.JsonClass
 
-actual typealias Json = Json
+@JsonClass(generateAdapter = true)
+actual data class GCSymmetricKey(val value: SecretKey)
```

Comment:
Note: This is now a data class now, since the normal class gave me a hard time for no obvious reason. I like to ensure that you are aware of this change.